account_customer_wallet/models/account_move.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Coop IT Easy SC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _post(self, soft=True):
+        res = super()._post(soft=soft)
+        if not soft:
+            partners = self.mapped("line_ids.partner_id")
+            partners |= partners.mapped("child_ids")
+            partners |= partners.mapped("parent_id")
```

Comment:
Both of these should be recursive.

Reply:
Done in the last commit. replaced by get_all_partners_in_family